pennylane/ops/qubit/hamiltonian.py

```diff
@@ -550,3 +550,80 @@ def queue(self, context=qml.QueuingContext):
 
         context.append(self, owns=tuple(self.ops))
         return self
+
+    def sparse_matrix(self, wires=None):
```

Comment:
Do we keep the utils.sparse_hamiltonian after having this functionality?

Reply:
Good question. What do you think, is there any advantage in keeping it?